filebeat/harvester/config.go

```diff
@@ -20,8 +21,10 @@ var (
 		BackoffFactor:   2,
 		MaxBackoff:      10 * time.Second,
 		CloseOlder:      1 * time.Hour,
-		ForceCloseFiles: false,
 		MaxBytes:        10 * (1 << 20), // 10MB
+		CloseRemoved:    false,
+		CloseRenamed:    false,
+		ForceCloseFiles: false,
```

Comment:
maybe we want to add a 'deprecated' option to go-ucfg, which will print a warning when option is used.

We can find deprecated options in code by grepping for `config:.*deprecated`. Maybe even with version like: `config:",deprecated(5.0)"`.